.editorconfig

```diff
@@ -0,0 +1,98 @@
+################################################
```

Comment:
I would remove this header from all files

Reply:
Done!